app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -0,0 +1,98 @@
+<ng-form name="persistentVolumeClaimForm">
+  <fieldset ng-disabled="routingDisabled">
+
+    <!-- Name -->
+    <div ng-show="showNameInput" class="form-group">
+      <label for="claim-name" class="required">Name</label>
+      <!-- regex, maxlength from k8s.io/kubernetes/pkg/util/validation/validation.go -->
+      <input
+        id="claim-name"
+        class="form-control"
+        type="text"
+        name="name"
+        ng-model="claim.name"
+        ng-required="showNameInput"
+        ng-pattern="/^[a-z0-9]([-a-z0-9]*[a-z0-9])?$/"
+        ng-maxlength="63"
+        ng-minlength="2"
+        placeholder="my-claim-name"
+        select-on-focus
+        autocorrect="off"
+        autocapitalize="off"
+        spellcheck="false"
+        aria-describedby="claim-name-help">
+      <div>
+        <span id="claim-name-help" class="help-block">A unique name for the storage claim within the project.</span>
+      </div>
+      <div class="has-error" ng-show="persistentVolumeClaimForm.name.$error.pattern && persistentVolumeClaimForm.name.$touched && !routingDisabled">
+        <span class="help-block">
+          Claim names may only contain lower-case letters, numbers, and dashes.
+          They may not start or end with a dash.
+        </span>
+      </div>
+    </div>
+
+    <div class="form-group" >
+      <label class="required">Access Mode</label><br/>
+      <!--modes = { 'RWX':'Read/Write Many', 'RWO':'Read/Write Once', 'ROX':'Read-Only Many' };-->
+      <div class="radio" ng-app ng-init="claim.accessModes=ReadWriteOnce">
+        <!--<div ng-init="claim.accessModes=ReadWriteOnce" />-->
+
+
+        <label class="radio-inline">
+          <fieldset>
+            <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce" aria-describedby="service-help" ng-checked="true" >
+            Single User (RWO)
+          </fieldset>
+        </label>
+
+        <label class="radio-inline">
+          <fieldset><input type="radio" id="accessModes" name="accessModes" ng-model="claim.accessModes" value="ReadWriteMany" aria-describedby="service-help" >
+            Shared Access (RWX)
+          </fieldset>
+        </label>
+
+        <label class="radio-inline">
+          <fieldset>
+            <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadOnlyMany" aria-describedby="service-help">
+            Read Only (ROX)
+          </fieldset>
+        </label>
+
+      </div>
+      <div>
+        <span id="service-help" class="help-block">Accessibility permissions to the mounted volume.</span>
+      </div>
+    </div>
+
+    <!-- capacity -->
+    <div class="form-group">
+      <label for="capacity" class="required">Capacity</label>
+      <!-- regex, maxlength from k8s.io/kubernetes/pkg/util/validation/validation.go -->
+      <input
```

Comment:
Rather than just a plain input, I'd rather have a text input for the number with a select for the units. We do this for edit resource limits elsewhere. See

https://github.com/openshift/origin-web-console/blob/master/app/views/_compute-resource.html#L3-L19

and

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/editRequestLimit.js#L67-L83

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/16989516/7b7871fc-4e62-11e6-865c-b5664dcf5dc7.png)